.github/workflows/create-nightly-tags.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+# Tag format example: v1.2.3-rc-1025201530
+create_tag_for_branch() {
+    local branch_name=$1
+    
+    git checkout "$branch_name"
+    
+    PKG_VERSION=$(cat ./package.json | grep 'version":' | sed 's/.*"version":[ \t]*"\([^"]*\)".*/\1/')
+    COMMIT_DATE=$(git log -1 --format=%cd --date=format:%m%d%H%M)
+    
+    if [[ -n "$PKG_VERSION" && -n "$COMMIT_DATE" ]]; then
+        TAG_NAME="v$PKG_VERSION-$COMMIT_DATE" 
+        echo "Creating tag for $branch_name: $TAG_NAME"
+        git tag "$TAG_NAME"
+        git push origin "$TAG_NAME"
+    else
+        echo "Failed to generate tag for $branch_name (PKG_VERSION: '$PKG_VERSION', COMMIT_DATE: '$COMMIT_DATE')"
```

Comment:
This will fail if tag already exists, would be nice if we actually handled that a bit better, maybe can be part of the telegram notificaiton PR, as that will need to only send the message if there's a new tag version.

Reply:
Yup thought of that too, handling it in later pr